protected/views/layouts/main.php

```diff
@@ -20,8 +20,109 @@
     <?= $this->renderPartial('//shared/_google_analytics')?>
 
     <title><?php echo CHtml::encode($this->pageTitle); ?></title>
-            <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/2.2.4/jquery.min.js"></script>
-        <script src="https://cdnjs.cloudflare.com/ajax/libs/jqueryui/1.12.1/jquery-ui.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/2.2.4/jquery.min.js"></script>
```

Comment:
@lump1k To improve the message for this [commit](dbb81e0be1ff4395f530ba5bcaef22bb9400faae), can you explain the purpose of this new CSS code please?

Reply:
.popover-content
http://i.imgur.com/yqzeoXU.png
for this popup. because it looked ajwul
select.dropdown-white
selects were grey and i made them white like other fields
#author-table td
removes breaks in author table
.additional-bordered
for divs with borders in additional tab
.additional-button
 .others-button
.others-input
 .others-label 
for styling some elements in additional tab
.js-not-allowed
for grey buttons which cant be clicked
.js-delete-column
.js-delete-row
#samples-table [type="text"]
.btn-sample
.sample-attribute-column
need it for sample tab

Reply:
@jessesiu can you confirm that this CSS code for the dataset submission wizard is best placed here in the [main.php](https://github.com/lump1k/gigadb-website/blob/vitaly/protected/views/layouts/main.php) file? Or should it be in common.css?

Reply:
Hi @lump1k @pli888 @only1chunts 

The submission wizard should use the layout [new_main.php](https://github.com/lump1k/gigadb-website/blob/vitaly/protected/views/layouts/new_main.php) , so this CSS code should put in the [common.css file ](https://github.com/lump1k/gigadb-website/blob/vitaly/css/common.css). So it can be reused for other layouts or pages in the common.css file.

Thanks,
Jesse

Reply:
yes, it will be better to add it to common.css. but can i do it in the end of all my tasks? because i dont want to do build all the time

Reply:
@lump1k OK, do it at the end of your tasks. Once this has been done then the message for this commit could be:

>Create Choose view for dataset submission wizard

>This commit renders web page to allow user to upload dataset metadata by spreadsheet or submission wizard. CSS code has also been added into common.css to improve the display of pop-up boxes, etc.

>Resolves WL Sprint 1 task: https://jirasoft.ffflabel-dev.com/browse/GIG-2 for issue #241